lib/src/main/res/layout/com_auth0_lock_btn_social_large.xml

```diff
@@ -43,8 +43,8 @@
         <TextView
             android:id="@+id/com_auth0_lock_text"
             style="@style/Lock.Theme.Text.SocialButton"
-            tools:background="#90FF3333"
-            tools:text="Login with facebook" />
+            tools:background="#FF3333"
```

Comment:
"tools" prefix indicates this change is only shown locally in the Layout Preview section, for testing purposes from inside the IDE.